taxonomy/utils.py

```diff
@@ -130,7 +142,42 @@ def is_course_skill_black_listed(course_key, skill_id):
     Returns:
         (bool): True if course-skill (identified by the arguments) is black-listed, False otherwise.
     """
-    return BlacklistedCourseSkill.objects.filter(
+    return CourseSkills.objects.filter(
         course_id=course_key,
-        skill_id=skill_id
+        skill_id=skill_id,
+        is_blacklisted=True,
     ).exists()
+
+
+def get_course_skills(course_key, prefetch_skills=True):
```

Comment:
The name of this method can be changed to `get_whitelisted_course_skills`

Reply:
Good suggestion, `get_whitelisted_course_skills` seems better. Done